packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/trust-store.ts

```diff
@@ -0,0 +1,120 @@
+import { Construct } from 'constructs';
+import { IBucket } from '../../../aws-s3';
+import { IResource, Resource, Fn, Names, Lazy } from '../../../core';
+import { CfnTrustStore } from '../elasticloadbalancingv2.generated';
+
+/**
+ * Represents a Trust Store
+ */
+export interface ITrustStore extends IResource {
+  /**
+   * The name of the trust store
+   * @attribute
+   */
+  readonly trustStoreName: string;
+
+  /**
+   * The ARN of the trust store
+   * @attribute
+   */
+  readonly trustStoreArn: string;
+}
+
+/**
+ * properties used for the Trust Store
+ */
+export interface TrustStoreProps {
+
+  /**
+   * The name of the trust store
+   *
+   * @default - Auto generated
+   */
+  readonly trustStoreName?: string;
+
+  /**
+   * The bucket that the trust store is hosted in
+   */
+  readonly bucket: IBucket;
+
+  /**
+   * The key in S3 to look at for the trust store
+   */
+  readonly key: string;
```

Comment:
In [CFn document](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-truststore.html#aws-resource-elasticloadbalancingv2-truststore-properties), this property is optional.

However, deploying without setting it resulted in an error, so I made it required.